README.md

```
@@ -250,6 +250,69 @@ generated from workouts are accepted without issues.
+                'messages': None
+            }
+        ],
+        'failures': []
+    }
+}
+```
+
+## Renaming an activity
+
+Using the "internal activity id" from above, an activity can be renamed using the
+`rename` method. This snippet shows an example of uploading an activity, fetching its
```

Markdownlint (0.37.0)

Expected: 80; Actual: 85
Line length

(MD013, line-length)